.ci/install-clang-devel.sh

```diff
@@ -32,26 +32,26 @@ add-apt-repository -y "deb-src http://apt.llvm.org/unstable/ llvm-toolchain-${CL
 apt-get update -y
 
 apt-get install -y --no-install-recommends \
-    clang-${CLANG_VERSION} \
-    clangd-${CLANG_VERSION} \
-    clang-format-${CLANG_VERSION} \
-    clang-tidy-${CLANG_VERSION} \
-    clang-tools-${CLANG_VERSION} \
-    lldb-${CLANG_VERSION} \
-    lld-${CLANG_VERSION} \
-    llvm-${CLANG_VERSION}-dev \
-    llvm-${CLANG_VERSION}-tools \
-    libomp-${CLANG_VERSION}-dev \
-    libc++-${CLANG_VERSION}-dev \
-    libc++abi-${CLANG_VERSION}-dev \
-    libclang-common-${CLANG_VERSION}-dev \
-    libclang-${CLANG_VERSION}-dev \
-    libclang-cpp${CLANG_VERSION}-dev \
-    libunwind-${CLANG_VERSION}-dev
+    clang-"${CLANG_VERSION}" \
```

Comment:
```suggestion
    "clang-${CLANG_VERSION}" \
```

Everywhere in this pull request where you've put double quotes in the middle of a word, please instead only used them around the entire word, like this example.

Reply:
Sure, I've changed the double quotes in .ci/install_clang_devel.sh file.